mlflow/entities/trace_location.py

```diff
@@ -69,11 +69,6 @@ class UCSchemaLocation(_MlflowObject):
     catalog_name: str
     schema_name: str
 
-    def to_proto(self):
-        return pb.TraceLocation.UCSchemaLocation(
-            catalog_name=self.catalog_name, schema_name=self.schema_name
-        )
-
     @classmethod
     def from_proto(cls, proto) -> "UCSchemaLocation":
         return cls(catalog_name=proto.catalog_name, schema_name=proto.schema_name)
```

Comment:
@serena-ruan Can we rename to `DatabricksUCSchemaLocation` and `DatabricksInferenceTableLocation`? Feel free to do this in a followup PR, but I want to make it obvious from the name that these belng to Databricks